test/reporter-html/index.html

```diff
@@ -3,7 +3,7 @@
 <head>
 	<meta charset="UTF-8">
 	<title>QUnit HTML Reporter Test Suite</title>
-	<link rel="stylesheet" href="../dist/qunit.css">
+	<link rel="stylesheet" href="../../dist/qunit.css">
```

Comment:
You don't need to remove that. That was something I didn't notice before because I'm running tests only from the cli (grunt and browserstack). 



> On Jun 17, 2015, at 8:36 PM, Kevin Partington <notifications@github.com> wrote:
> 
> In test/reporter-html/index.html:
> 
> > @@ -3,7 +3,7 @@
> >  <head>
> >  	<meta charset="UTF-8">
> >  	<title>QUnit HTML Reporter Test Suite</title>
> > -	<link rel="stylesheet" href="../dist/qunit.css">
> > +	<link rel="stylesheet" href="../../dist/qunit.css">
> Do you want me to remove that? I couldn't run the QUnit HTML Reporter suite without that change.
> 
> —
> Reply to this email directly or view it on GitHub.
>